Include 'Isolated' App Service Plan

andynava opened the pull request:
When removing instances, this is considered "scale in" not "scale down"
It doesn't count when the function is "running", but the number of times it is "executed" (Invoked)

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

andynava committed:
Include 'Isolated' App Service Plan

When removing instances, this is considered "scale in" not "scale down"
It doesn't count when the function is "running", but th number of times it is "executed" (Invoked)
